<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>proftp</class>
 <widget class="QMainWindow" name="proftp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>proftp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="windowHome" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QPushButton" name="buttonServerManager">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Server manager</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="logsFTP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>411</width>
       <height>61</height>
      </rect>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="serversSelect">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonConnectServer">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QWidget" name="windowDataTransfert" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>431</width>
       <height>381</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Remote files :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonReturnDirectory">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddFolderRemote">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDeleteFolderRemote">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTreeWidget" name="remoteFolderView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>301</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Owner</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Group</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="buttonDownload">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Local files :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAddFolderLocal">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonDeleteFolderLocal">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTreeView" name="localFolderView">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonUpload">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QListWidget" name="localFilesView">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>201</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonSync">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sync</string>
     </property>
    </widget>
    <widget class="QWidget" name="windowSync" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>431</width>
       <height>391</height>
      </rect>
     </property>
     <widget class="QPushButton" name="buttonSynchroniseFolders">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Synchronise folders</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Remote files :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Local files :</string>
      </property>
     </widget>
     <widget class="QListWidget" name="localFilesSync">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>201</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="remoteFilesSync">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="windowServerManager" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QPushButton" name="buttonHome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="addServerEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddServer">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add this server</string>
     </property>
    </widget>
    <widget class="QListWidget" name="serversList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="informationsServerBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>261</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Informations</string>
     </property>
     <widget class="QPushButton" name="buttonDeleteServers">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>13</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../ProgFTP-build-desktop-Qt_4_8_1_for_Desktop_-_MinGW__Qt_SDK__Debug/images/notsync.png</normaloff>../ProgFTP-build-desktop-Qt_4_8_1_for_Desktop_-_MinGW__Qt_SDK__Debug/images/notsync.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom du server :</string>
      </property>
     </widget>
     <widget class="QLabel" name="serverNameLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverAdressEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Login :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverLoginEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverPasswordEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Port :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverPortEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Local folder :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonChangeLocalFolder">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Remote folder :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonChangeRemoteFolder">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QLabel" name="localFolderLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="remoteFolderLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="propertiesBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>400</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Properties</string>
     </property>
     <widget class="QCheckBox" name="autoLoginCheck">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto-login with</string>
      </property>
     </widget>
     <widget class="QComboBox" name="serversSelectProperties">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="windowStart" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>111</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>More</string>
     </property>
     <widget class="QPushButton" name="buttonForum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Forum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonHelp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAboutProgFTP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>About ProgFTP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAboutQt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>About Qt</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonGoToProgFTP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to ProgFTP and add FTP server</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addNewServerEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>261</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Bienvenue sur ProgFTP,&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;ce programme va vous permettre de synchroniser des dossiers à partir d'un serveur FTP sur votre ordinateur. Vous pouvez aussi télécharger indépendandement des fichiers et les uploader. Avant de commencer, vous êtes obligé de créer un serveur FTP.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Il existe de nombreux autres programmes de synchronisation de dossiers entre un serveur FTP et votre ordinateur, mais celui-ci est très léger et simple d'utilisation. Il sera régulièrement mis à jour par son créateur, Dy006. En vous remerciant pour votre participation, ProgAzur.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrer le nom de votre serveur :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ou</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter le fichier de configuration du serveur au répertoire</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;servers/&quot; du programme pluis cliquer sur ce bouton :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRefreshServeurs">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionServer_manager"/>
    <addaction name="separator"/>
    <addaction name="actionLogs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSynchronise_folders"/>
    <addaction name="actionDownload_file"/>
    <addaction name="actionUpload_file"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_ProgFTP"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuServer"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLogs">
   <property name="text">
    <string>Logs</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionServer_manager">
   <property name="text">
    <string>Server manager</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSynchronise_folders">
   <property name="text">
    <string>Synchronise folders</string>
   </property>
  </action>
  <action name="actionDownload_file">
   <property name="text">
    <string>Download file</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_ProgFTP">
   <property name="text">
    <string>About ProgFTP</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionUpload_file">
   <property name="text">
    <string>Upload file</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
